android/app/src/main/AndroidManifest.xml

```diff
@@ -1,60 +1,78 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-		  xmlns:tools="http://schemas.android.com/tools"
-	package="chat.rocket.reactnative">
-
-	<uses-permission android:name="android.permission.INTERNET" />
-	<uses-permission android:name="android.permission.CAMERA" />
-	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-	<uses-permission android:name="android.permission.RECORD_AUDIO" />
-	<uses-permission android:name="android.permission.WAKE_LOCK" />
-	<uses-permission android:name="android.permission.DOWNLOAD_WITHOUT_NOTIFICATION" />
-	<uses-permission-sdk-23 android:name="android.permission.VIBRATE"/>
-
-	<application
-		android:name=".MainApplication"
-		android:allowBackup="true"
-		android:label="@string/app_name"
-		android:icon="@mipmap/ic_launcher"
-		android:theme="@style/AppTheme"
-	>
-		<activity
-			android:name=".MainActivity"
-			android:label="@string/app_name"
-			android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-			android:launchMode="singleTask"
-			android:windowSoftInputMode="adjustResize">
-			<intent-filter>
-				<action android:name="android.intent.action.MAIN" />
-				<category android:name="android.intent.category.LAUNCHER" />
-				<action android:name="android.intent.action.DOWNLOAD_COMPLETE"/>
-			</intent-filter>
-			<intent-filter android:label="@string/app_name">
-				<action android:name="android.intent.action.VIEW" />
-				<category android:name="android.intent.category.DEFAULT" />
-				<category android:name="android.intent.category.BROWSABLE" />
-				<data android:scheme="https" android:host="go.rocket.chat" />
-				<data android:scheme="rocketchat" android:host="room" />
-				<data android:scheme="rocketchat" android:host="auth" />
-			</intent-filter>
-		</activity>
-		<activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-		<activity
-				android:noHistory="true"
-				android:name=".share.ShareActivity"
-				android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-				android:label="@string/share_extension_name"
-				android:screenOrientation="portrait"
-				android:theme="@style/AppTheme" >
-				<intent-filter>
-					<action android:name="android.intent.action.SEND" />
-					<category android:name="android.intent.category.DEFAULT" />
-					<data android:mimeType="*/*" />
-				</intent-filter>
-		</activity>
-		<meta-data
-				android:name="com.bugsnag.android.API_KEY"
-				android:value="${BugsnagAPIKey}" />
-	</application>
+    xmlns:tools="http://schemas.android.com/tools"
+    package="chat.rocket.android"
+    android:sharedUserId="chat.rocket.android">
+
+    <application
```

Comment:
This refactor is not needed

Reply:
Copying and pasting what we wrote above:

Sorry, but is was a small change and since the PR objective touched on it I believe we don't have any problems on doing that! Any improvement, more yet when it is very small, is always welcome on any PR and in a such codebase that can leads us to forget it later.